includes/installation/additional_ipv6.md

```diff
@@ -3,15 +3,46 @@ Sofern eine eigene IPv6 Adresse für diesen Dienst verwendet werden soll,
 wird diese der entsprechenden Netzwerkschnittstelle hinzugefügt, sodass 
 diese in nginx verwendet werden kann. 
 
-```shell
-# /etc/network/interfaces
+=== "Debian"
+    ```shell
+    # /etc/network/interfaces
+    
+    # ...
+    
+    iface eth0 inet6 static
+        # ipv6 address of the host
+        address 2001:db8:1234:5678::1/64
+        gateway 2001:db8::1
+        # service.domain.de
+        post-up ip -6 a add 2001:db8:1234:5678:5eca:dc9d:fd4e:6564/64 dev eth0
+    ```
 
-# ...
-
-iface eth0 inet6 static
-    # ipv6 address of the host
-    address 2001:db8:1234:5678::1/64
-    gateway 2001:db8::1
-    # service.domain.de
-    post-up ip -6 a add 2001:db8:1234:5678:5eca:dc9d:fd4e:6564/64 dev eth0
-```
\ No newline at end of file
+=== "Ubuntu"
+    Unter Ubuntu ist es etwas anders. Von Ubuntu aus hat man in `/etc/netplan` eine `50-cloud-init.yaml`, wenn diese nicht
```

Comment:
~~Unter Ubuntu ist es etwas anders. Von Ubuntu aus hat man in `/etc/netplan` eine `50-cloud-init.yaml`, wenn diese nicht existiert, dann muss man schauen, welche Datei die Konfigurationsdatei für das Netzwerk ist.~~

- schlechter Ausdruck
- `50-cloud-init.yaml` ist Hosterspezifisch -> hier: netcup

Da Ubuntu `netplan` zum Konfigurieren der Netzwerkeschnittstellen verwendet, muss die entsprechende Konfiguration im  Verzeichnis `/etc/netplan` angepasst werden.